testrunner-running/src/main/scala/edu/illinois/cs/testrunner/execution/JUnitTestRunner.java

```diff
@@ -57,6 +57,22 @@ public static String fullName(final Description description) {
         return methods;
     }
 
+    private static Object testObj;
+
+    public static Object getTestObj() {
```

Comment:
Did we end up actually using these changes for the `JUnitTestRunner`? I recall we were trying to bypass something concerning JUnit, but I don't remember if it ended up being necessary...